src/status_im2/contexts/wallet/collectible/view.cljs

```diff
@@ -57,25 +58,26 @@
 
 (defn traits-section
   [traits]
-  [rn/view {:style style/traits-section}
-   [quo/section-label
-    {:section (i18n/label :t/traits)}]]
-
-  [rn/flat-list
-   {:render-fn               (fn [{:keys [title subtitle]}]
-                               [rn/view {:style style/traits-item}
-                                [quo/data-item
-                                 {:description :default
-                                  :card?       true
-                                  :status      :default
-                                  :size        :default
-                                  :title       title
-                                  :subtitle    subtitle}]])
-    :data                    traits
-    :key                     :collectibles-list
-    :key-fn                  :id
-    :num-columns             2
-    :content-container-style style/traits-container}])
+  (when (pos? (count traits))
+    [rn/view
+     [quo/section-label
+      {:section         (i18n/label :t/traits)
+       :container-style style/traits-title-container}]
+     [rn/flat-list
+      {:render-fn               (fn [{:keys [trait-type value]}]
+                                  [rn/view {:style style/traits-item}
+                                   [quo/data-item
```

Comment:
I think you can make use of container-style prop on data-item component 
```
(fn [{:keys [trait-type value]}]
                                   [quo/data-item
                                    {:container-style style/traits-item
                                     :description :default
                                     :card?       true
                                     :status      :default
                                     ...
 ```
 
 it should already exist 👍

Reply:
@J-Son89, thanks, good point!